pkg/indexeres/indexer.go

```diff
@@ -68,3 +77,25 @@ func VMBackupBySourceVMUID(obj *harvesterv1.VirtualMachineBackup) ([]string, err
 func VMBackupBySourceVMName(obj *harvesterv1.VirtualMachineBackup) ([]string, error) {
 	return []string{obj.Spec.Source.Name}, nil
 }
+
+func VMTemplateVersionByImageId(obj *harvesterv1.VirtualMachineTemplateVersion) ([]string, error) {
+	volumeClaimTemplateStr, ok := obj.Spec.VM.ObjectMeta.Annotations[util.AnnotationVolumeClaimTemplates]
+	if !ok || volumeClaimTemplateStr == "" {
+		return []string{}, nil
+	}
+
+	imageIds := []string{}
```

Comment:
It's better to use `var imageIds []string` instead

Reply:
Updated it. Thanks.

Reply:
If I use `var imageIds []string`, lint will show me `consider pre-allocating (prealloc)`. However, I can't preallocate length here, because we may skip some `imageID` here.